tests/reset/reset.cpp

```diff
@@ -268,6 +269,12 @@ TEST(Reset, addRemoveResetFromComponentMethods)
     r2->setTestValue(EMPTY_MATH);
     c->addReset(r2);
 
+    // Try and add an existing reset to trigger 'false' return
```

Comment:
Missing full stop at the end of this and other comments.

Reply:
d9b0052d